server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -133,45 +138,92 @@ private FormTag buildProgramForm(
     FormTag formTag = form().withMethod("POST").withId("program-details-form");
     formTag.with(
         requiredFieldsExplanationContent(),
-        h2("Visible to applicants").withClasses("py-2", "mt-6", "font-semibold"),
+        h2("Program setup").withClasses("py-2", "mt-6", "font-semibold"),
         FieldWithLabel.input()
             .setId("program-display-name-input")
             .setFieldName("localizedDisplayName")
-            .setLabelText("Enter the publicly displayed name for this program")
+            .setLabelText("Program name")
             .setRequired(true)
             .setValue(displayName)
             .getInputTag(),
+        insertSpacingDiv(),
+        AlertComponent.renderSlimAlert(
+            AlertType.INFO,
+            "Short description will be visible to applicants at a future date.",
+            false,
+            "my-2"),
         FieldWithLabel.textArea()
-            .setId("program-display-description-textarea")
-            .setFieldName("localizedDisplayDescription")
-            .setLabelText("Describe this program for the public")
+            .setId("program-display-short-description-textarea")
+            .setFieldName("localizedShortDescription")
+            .setLabelText(
+                "Short description of this program for the public. Maximum 100 characters.")
```

Comment:
**question (non-blocking):** This label text doesn't match the mocks.  Is that intentional?

Reply:
I like the label you made.  Seems good until we add the headings.